specs/wp-view-site-spec.js

```diff
@@ -0,0 +1,71 @@
+import assert from 'assert';
+import test from 'selenium-webdriver/testing';
+
+import config from 'config';
+import * as driverManager from '../lib/driver-manager.js';
+
+import MySitePage from '../lib/pages/mysite-page';
+import SidebarComponent from '../lib/components/sidebar-component.js';
+import LoginFlow from '../lib/flows/login-flow.js';
+
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+const screenSize = driverManager.currentScreenSize();
+
+var driver;
+
+test.before( function() {
+	this.timeout( startBrowserTimeoutMS );
+	driver = driverManager.startBrowser();
+} );
+
+test.describe( 'View site from sidebar: (' + screenSize + ')', function() {
+	this.timeout( mochaTimeOut );
+	this.bailSuite( true );
+
+	test.describe( 'View site and close:', function() {
+		test.before( function() {
+			driverManager.clearCookiesAndDeleteLocalStorage( driver );
+		} );
+
+		test.it( 'Can Log In and go to My Sites', function() {
+			const loginFlow = new LoginFlow( driver );
+			loginFlow.loginAndSelectMySite();
```

Comment:
not every spec has it currently, but we try to use an explicit return statement for the last line of each `test.it` block to ensure the WebdriverJs promise manager works as expected.

So this would be something like:

`return loginFlow.loginAndSelectMySite();`